lua/terrortown/menus/gamemode/equipment/base_equipment.lua

```diff
@@ -99,4 +99,14 @@ function CLGAMEMODESUBMENU:Populate(parent)
 
 		masterRefs[key] = option
 	end
+
+	-- now add custom equipment settings
+	if not isfunction(equipment.AddToSettingsMenu) then
```

Comment:
isn't this true ("false" here) then for all weapons / items (that use the ttt base), because the function is defined on the base as an empty function?

Reply:
This PR is WIP and this is just for debugging